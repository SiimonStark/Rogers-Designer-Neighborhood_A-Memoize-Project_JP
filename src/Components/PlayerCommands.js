import React, { Component } from 'react';
import GridInput from './GridInput';
import FlexInput from './FlexInput';

class PlayerCommands extends Component {
  constructor(props) {
    super(props)
    this.state = {
    }
  }
  render() {
    return (
      <section className="PlayerCommands">
        <GridInput updateInputs={this.props.updateInputs} />
        <FlexInput updateInputs={this.props.updateInputs} />
      </section>
    );
  }
}

export default PlayerCommands;