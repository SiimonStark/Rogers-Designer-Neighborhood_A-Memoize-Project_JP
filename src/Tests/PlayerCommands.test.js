import React from 'react';
import PlayerCommands from '../Components/PlayerCommands';
import { shallow } from 'enzyme';
import PropTypes from "prop-types";

let updateInputsMock = jest.fn();

describe('PlayerCommands', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<PlayerCommands updateInputs={updateInputsMock} />);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it.skip('should have props method to pass on', () => {
    console.log(wrapper.props('updateInputs'))
    expect(wrapper.props()).toEqual('updateInputs');
  });
});