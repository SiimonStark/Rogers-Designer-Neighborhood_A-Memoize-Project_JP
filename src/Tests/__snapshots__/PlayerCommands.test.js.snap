// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerCommands should match snapshot 1`] = `
<section
  className="PlayerCommands"
>
  <GridInput
    updateInputs={[MockFunction]}
  />
  <FlexInput
    updateInputs={[MockFunction]}
  />
</section>
`;
